app/views/wpcc/your_details/new.html.erb

```diff
@@ -123,6 +123,16 @@
                 <p><%= t('wpcc.details.callout__gt5876_lt10000') %></p>
               </div>
             </div>
+            <div class="form__row details__callout details__callout--inactive" data-wpcc-callout-near_pension_threshold>
+              <div class="callout callout--warning">
+                <p><%= t('wpcc.details.near_pension_limit_message_html') %></p>
+              </div>
+            </div>
+            <div class="form__row details__callout details__callout--inactive" data-wpcc-callout-near_auto_enrollment_threshold>
+              <div class="callout callout--warning">
+                <p><%= t('wpcc.details.near_manual_opt_in_limit_message_html') %></p>
+              </div>
+            </div>
```

Comment:
The code itself is fine, but this has become unwieldy and I'm wondering if it should be refactored, even if its just splitting out into some partials?

Reply:
I agree with this comment. I would like to address that refactor in another PR as that may be a large chunk of changes. Given this PR is going into a feature branch, I don't want to increase the size of this feature branch too much making it harder for review

Reply:
Please create a tech debt card and we can address later.